app/Actions/PhotoAuthorisationProvider.php

```diff
@@ -304,9 +306,9 @@ public function areEditableByIDs(array $photoIDs): bool
 		return
 			count($photoIDs) === 0 ||
 			Photo::query()
-				->whereIn('id', $photoIDs)
-				->where('owner_id', $user->id)
-				->count() === count($photoIDs);
+			->whereIn('id', $photoIDs)
```

Comment:
@ildyria Here we have another file where the indention has been changed to the worse since I had reviewed it the last time. Please check with your IDE whether you have some option which changes the indention upon opening/saving. It is not cs-fixer.